cristin-organization/src/main/java/no/unit/nva/cristin/organization/fetch/DefaultOrgFetchClientProvider.java

```diff
@@ -0,0 +1,37 @@
+package no.unit.nva.cristin.organization.fetch;
+
+import static no.unit.nva.cristin.organization.common.Constants.CLIENT_WANTS_VERSION_OF_THE_API_CLIENT;
+import static no.unit.nva.cristin.organization.common.Constants.VERSION_2023_05_26;
+import static no.unit.nva.cristin.organization.common.Constants.VERSION_ONE;
+import java.util.Map;
+import no.unit.nva.cristin.common.client.ClientProvider;
+import no.unit.nva.cristin.common.client.FetchApiClient;
+import no.unit.nva.cristin.organization.common.client.CristinOrganizationApiClient;
+import no.unit.nva.cristin.organization.common.client.version20230526.CristinOrgApiClient20230526;
+import no.unit.nva.model.Organization;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultOrgFetchClientProvider
+    implements ClientProvider<FetchApiClient<Map<String, String>, Organization>> {
+
+    private static final Logger logger = LoggerFactory.getLogger(DefaultOrgFetchClientProvider.class);
+
+    @Override
+    public FetchApiClient<Map<String, String>, Organization> getClient(String apiVersion) {
+        if (VERSION_2023_05_26.equals(apiVersion)) {
```

Comment:
What if a client ask for version 2023_06_19, should we return first or last prior version?

Reply:
defaults then to first version